.wordlist.txt

```diff
@@ -1,18 +1,40 @@
+addons
+API
+APIs
+balancer
+CharmHub
+CLI
+dropdown
 Diátaxis
+EBS
+EKS
 favicon
-https
+filesystem
```

Comment:
```suggestion
filesystem
```
That's probably up to each project to decide, but for LXD I insist on "file system" - two words.